### ISSUE TITLE
"File not working"

### ISSUE THREAD

**Issue by phelipe** [codeneomatrix/Merly.jl#13]
**Describe the bug**
@page "/" File("Index.html")   is not working, it trows
```
┌ Error: error handling request
│   exception =
│    UndefVarError: File not defined

```
**To Reproduce**
1. Set up environment
2. Arguments
3. Function/method

**Expected behavior**
Load the html page

**Environment**
 - Repository
 - OS: Manaro Linux
 - REPL

**Comment by codeneomatrix** [comment]
Thanks for your report. To help, just answer a few questions:
- What version of julia are you using?
- Is the location of your "Index.html" file the same as that of your program? (https://github.com/codeneomatrix/Merly.jl/tree/5f3d1feac449134f0618a59317b60e6c83f8993a/test)

**Comment by phelipe** [comment]
Julia 1.4.2, and Yes. The error only trows when I try access the url.

**Comment by codeneomatrix** [comment]
Thank you very much, I will review the problem and try to solve it as soon as possible.

**Comment by phelipe** [comment]
Hi! I fixed this bug and other minors, but I had to modify the struct "App".  The struct was using functions inside, this was making  all files get public in julia > 1.0 even when set webserverpath and webserverfiles. The api will work like this:

```
ip = "127.0.0.1"
port = 8086
server = App(ip, port) 

webserverpath(server, "cosa")
webserverfiles(server, "*")
@page "/mifile" File(server, "Index.html")

start(server, verbose = false)
``` 
I don't know if you will like this modification, I will make a PR soon for you see the modifications.
